nemo/core/classes/modelPT.py

```diff
@@ -544,6 +544,11 @@ def setup_optimization(self, optim_config: Optional[Union[DictConfig, Dict]] = N
         if lr is not None:
             optimizer_args['lr'] = lr
 
+        # Distributed optimizer needs process group for reductions
+        if optimizer_name == 'distributed_fused_adam':
```

Comment:
Core should not special-case something like this, nor import unguarded optional import like this. Please revert or move into nlp override function.